testing/transfer-eth-erc20/from-near.js

```diff
@@ -323,9 +338,9 @@ class TransferEthERC20FromNear {
           gasPrice: new BN(await robustWeb3.web3.eth.getGasPrice()).mul(new BN(ethGasMultiplier))
         }
       )
-      const newBalance = await ethERC20Contract.methods
-        .balanceOf(ethReceiverAddress)
-        .call()
+      // TODO investigate and fix this web3 call
+      // const newBalance = await ethERC20Contract.methods.balanceOf(ethReceiverAddress).call()
```

Comment:
What's about this TODO?

Reply:
There is an unknown reason why this call fail when loading the contract from the abi, I think this research is not urgent and could be fixed in a follow-up issue. So for now at least we should have a passed ci to detect buggy changes.

Reply:
Anyway, I think the `ethers` interface that you've used is better than loading the whole abi. Partial ABI is better, so please just remove this TODO then